cinema/views.py

```diff
@@ -1 +1,28 @@
-# write views here
+from rest_framework import viewsets
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.serializers import GenreSerializer, ActorSerializer, CinemaHallSerializer, MovieSerializer, MovieSessionSerializer
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
```

Comment:
According to the checklist, you should not use the `.all()` method before `.prefetch_related()` or `.select_related()` methods. Consider removing `.all()` if you plan to use these methods.